ecommerce/models.py

```diff
@@ -82,3 +82,14 @@ class Cart(models.Model):
 
     def get_total_price(self):
         return self.quantity*self.product.price
+
+class Comment(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
+    product = models.ForeignKey('Product', on_delete=models.CASCADE, null=True)
+    title = models.CharField(default='No title', max_length=500)
+    content = models.TextField(default='', max_length=1000)
+    rate = models.IntegerField(default=5, blank=True, null=True)
+    date = models.DateField(default=now, blank=True)
```

Comment:
Cái này tự động đi em
```suggestion
    date = models.DateField(auto_now_add=True)
```